### PR TITLE
EVG-6300: hide unresponsive modal on restart task failure

### PR DESCRIPTION
**Pull request by dominoweir**
When restarting a task fails, the modify task modal is closed and the error is displayed at the top of the window as a notification. This fix has been verified in staging.

### CODE REVIEW

File: public/static/js/task_admin.js [hunk 1]
```diff
@@ -38,6 +38,15 @@ mciModule.controller('AdminOptionsCtrl', ['$scope', '$window', '$rootScope', 'mc
         }
     }
 
+    function doModalFailure(message, data, reload){
```
Comment (john-m-liu) [inline public/static/js/task_admin.js]: What's the data parameter used for?

File: public/static/js/task_admin.js [hunk 2]
```diff
@@ -38,6 +38,15 @@ mciModule.controller('AdminOptionsCtrl', ['$scope', '$window', '$rootScope', 'mc
...
+        $scope.closeAdminModal();
+        if (reload) {
```
Comment (john-m-liu) [inline public/static/js/task_admin.js]: I don't think there's any need to reload on error. It's reloaded on success to update displays

File: public/static/js/task_admin.js (hunk 1 above)
Comment (dominoweir) [reply public/static/js/task_admin.js]: Nothing- I modeled this on the modal success function, which does use the data parameter, and forgot to remove it when I was done writing my function. I have now removed that parameter.

File: public/static/js/task_admin.js (hunk 2 above)
Comment (dominoweir) [reply public/static/js/task_admin.js]: You're right- I've cleaned up the function so that it doesn't include the reload logic anymore.

**Pull request merged**